urifind.go

```diff
@@ -183,7 +179,8 @@ func PDFTitleGet(url string) string {
 		}
 	}()
 
-	gTitle, _, gErr := TitleGet("https://webcache.googleusercontent.com/search?q=cache:" + url)
+	c := http.Client{Timeout: 10 * time.Second}
```

Comment:
Wieso wird der http-Client jedes Mal neu angelegt, und nicht wie vorher in TitleGet?